### PR TITLE
Update script for latest packages & add Docker support

### PR DESCRIPTION
**Pull request by vit1-irk**

### CODE REVIEW

File: tgmusicbot.py [hunk 1]
```diff
@@ -210,7 +211,7 @@ def make_squarethumb(thumbnail, output):
     original_thumb = Image.open(thumbnail)
     squarethumb = _crop_to_square(original_thumb)
     squarethumb.thumbnail((TG_THUMB_MAX_LENGTH, TG_THUMB_MAX_LENGTH),
-                          Image.ANTIALIAS)
+                          Image.Resampling.LANCZOS)
```
Comment (dashezup) [inline tgmusicbot.py:214]: What's the differences?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.10-alpine
+
+COPY requirements.txt ./requirements.txt
+COPY tgmusicbot.py ./tgmusicbot.py
+
+RUN python -m pip install -r requirements.txt
+RUN rm -rf /root/.cache/pip
+
+ENTRYPOINT [ "python", "tgmusicbot.py" ]
```
Comment (dashezup) [inline Dockerfile]: text files should have new line characters at the end of the file

Comment (dashezup) [reply Dockerfile]: docker-compose.yml has the same issue

File: tgmusicbot.py (hunk 1 above)
Comment (vit1-irk) [reply tgmusicbot.py:214]: Pillow says it's the same, just ANTIALIAS will be deprecated in future

File: Dockerfile (hunk 2 above)
Comment (vit1-irk) [reply Dockerfile]: fixed

**Pull request merged**